base/Siege.php

```diff
@@ -91,12 +91,10 @@ protected function getArguments(): Vector<string> {
     if (!$this->options->noTimeLimit) {
       $arguments = Vector {
         // See Siege::getExecutablePath()  - these arguments get passed to
-        // 'timeout'
-        '-k', '5m',
         '5m',
         parent::getExecutablePath(),
       };
-    };
+    }
```

Comment:
from offline discussion: --signal=9 is fine, because this only happens when it's already tried to kill itself, and is stuck in a signal handling loop